lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -92,7 +92,7 @@ describe('LMSFilePicker', () => {
     assert.isTrue(authButton.exists());
 
     // Click the "Authorize" button and check that files are re-fetched.
-    const expectedFiles = [];
+    const expectedFiles = ['a file'];
```

Comment:
I generally prefer that test data looks realistic in the sense that it has the same or roughly the same structure as real data would. I know there are other examples of tests in this file that also use strings to stand in for file objects.

Reply:
Hah. I had the same reaction when I saw the other tests in this file (that it would be nicer if they looked like `File` objects) but was following the established lead, like a sheep.